.github/workflows/weekly-dep-update.yaml

```
@@ -40,16 +37,17 @@ jobs:
-        with:
-          branch: chore/deps-${{ steps.date.outputs.yearweek }}
-          title: "chore: Weekly dependency update (${{ steps.date.outputs.yearweek }})"
-          body: |
-            This PR updates all dependencies to their latest versions.
-            Automatically generated by GitHub Actions.
-          labels: dependencies
-          base: main
-          commit-message: "chore: weekly dependency update (${{ steps.date.outputs.yearweek }})"
-          committer: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
-          author: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
+              if: steps.deps.outputs.has_changes == 'true'
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)